packages/plugin-rest-client/src/__tests__/AbstractEntitySpec.js

```diff
@@ -74,6 +74,7 @@ describe('AbstractEntity', () => {
     }
 
     beforeEach(() => {
+      // eslint-disable-next-line no-unused-vars
       serializeCalled = false;
```

Comment:
Do we actually need to define this variable? Can't we just delete this beforeEach script completely?